src/Illuminate/Testing/TestResponse.php

```diff
@@ -57,23 +65,26 @@ class TestResponse implements ArrayAccess
      * Create a new test response instance.
      *
      * @param  \Illuminate\Http\Response  $response
+     * @param  \Illuminate\Http\Request  $request
      * @return void
      */
-    public function __construct($response)
+    public function __construct($response, $request)
     {
         $this->baseResponse = $response;
+        $this->request = $request;
         $this->exceptions = new Collection;
     }
 
     /**
      * Create a new TestResponse from another response.
      *
      * @param  \Illuminate\Http\Response  $response
+     * @param  \Illuminate\Http\Request  $request
      * @return static
      */
-    public static function fromBaseResponse($response)
+    public static function fromBaseResponse($response, $request)
```

Comment:
This is a breaking change.

Reply:
Yes, but it's not used anywhere else in the framework.